src/modules/workSession/interfaces/IWorkSession.service.ts

```diff
@@ -16,4 +17,7 @@ export interface IWorkSessionService {
   updateActiveTask(
     updateActiveTaskServiceDto: UpdateActiveTaskServiceDto,
   ): Promise<WorkSession>;
+  getWorkSessionByUserId(
```

Comment:
It would look nicer if the name is `getWorkSessionsByUserId` (getWorkSessionByUserId).

Because...
- It returns multiple workSessions
- In the controller, the name of the corresponding method is `getWorkSessionsByUserId`. It's plural